### PR TITLE
Added a parameter to exlude specific file(s)

### PR DESCRIPTION
**Pull request by fabiocchetti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +85,12 @@ Default `.`
 
 This is the base directory for all of your SASS/SCSS files. Grunt will recurse this directory looking for color variables.
 
+#### excludedFiles
+Type: `String`;
+Default `/`
```
Comment (bvaughn) [inline README.md]: This default seems a bit ambiguous to me. Could be interpreted as "exclude everything under the root". Maybe we could just say "Default null" in the docs (regardless of what the actual underlying default is).

**Review by bvaughn — CHANGES_REQUESTED** [review]
In general, this seems like a reasonable change. Happy to approve it and make a new release- but I'd like to see the tests updated to cover this new functionality. (Basically add a file, then exclude it, then verify in the expected outcome that it's been properly excluded.)

**Comment by bvaughn** [comment]
Published 1.1.0

Thanks for the contribution

**Comment by fabiocchetti** [comment]
Thanks to you for your collaboration!

**Pull request merged**